service/controller/chart/resource/status/create.go

```diff
@@ -81,12 +82,14 @@ func (r *Resource) EnsureCreated(ctx context.Context, obj interface{}) error {
 		AppVersion: releaseContent.AppVersion,
 		Reason:     reason,
 		Release: v1alpha1.ChartStatusRelease{
-			LastDeployed: metav1.Time{Time: releaseContent.LastDeployed},
-			Revision:     releaseContent.Revision,
-			Status:       status,
+			Revision: to.IntP(releaseContent.Revision),
+			Status:   status,
 		},
 		Version: releaseContent.Version,
 	}
+	if !releaseContent.LastDeployed.IsZero() {
+		desiredStatus.Release.LastDeployed = &metav1.Time{Time: releaseContent.LastDeployed}
+	}
```

Comment:
LastDeployed became an optional field in the CRD. So we need to handle nils or the operator crashes.

Reply:
Ran into a problem in giraffe which is the CRD wasn't up to date and these optional fields were still required.

We should re-run `opsctl ensure crds` for the chart CRD before releasing this.

Reply:
Isn't that `chartcrd` resource in app-operator doing that? or is that resource only for TCs?